Code/client/Services/Generic/CharacterService.cpp

```diff
@@ -493,6 +513,14 @@ void CharacterService::OnCharacterSpawn(const CharacterSpawnRequest& acMessage)
     if (pActor->IsDead() != acMessage.IsDead)
         acMessage.IsDead ? pActor->Kill() : pActor->Respawn();
 
+    spdlog::info("Spawn Reqeust Is summon {} {}", acMessage.IsPlayerSummon, m_world.playerHandler ? "handle" : "no handle");
+
+    if (acMessage.IsPlayerSummon && m_world.playerHandler)
```

Comment:
This is flawed. If the local player never summoned anything, there'd be no "playerHandler".